bpfd/src/main.rs

```diff
@@ -118,10 +121,21 @@ fn has_cap(cset: caps::CapSet, cap: caps::Capability) {
 }
 
 fn drop_linux_capabilities() {
-    let res = User::from_uid(getuid()).unwrap().unwrap();
+    let uid = getuid();
+    let res = match User::from_uid(uid) {
+        Ok(res) => res.unwrap(),
+        Err(_e) => {
```

Comment:
I don't think we want to return error back to user. I could include it in the log. This was only hitting an error in a container. Kenjiro had a separate PR that suggested it was because the scratch image didn't have a /etc/passwd file. I'll revisit in the bpfd operator work and make sure there is a bpfd user.

Reply:
I think I had `Err(_)` and one of the link/format suggested the `_e`. I'll remove the `e` and see if I hit it again. Maybe not, can't remember.

Reply:
Threw a debug log in to capture the error string.